controllers/fenceagentsremediation_controller.go

```diff
@@ -168,10 +170,21 @@ func buildFenceAgentParams(far *v1alpha1.FenceAgentsRemediation) ([]string, erro
 		return nil, errors.New(errorMissingParams)
 	}
 	var fenceAgentParams []string
+	logger := ctrl.Log.WithName("build-fa-parameters")
+	// add shared parameters except the action parameter
 	for paramName, paramVal := range far.Spec.SharedParameters {
-		fenceAgentParams = appendParamToSlice(fenceAgentParams, paramName, paramVal)
+		if paramName == parameterActionName {
+			if paramVal != parameterActionValue {
+				logger.Info("FAR doesn't support any other action than reboot", "action", paramVal)
+			}
+		} else {
+			fenceAgentParams = appendParamToSlice(fenceAgentParams, paramName, paramVal)
+		}
```

Comment:
Nit: I think this can be simplified a bit into:
```
if paramName != parameterActionName {
	fenceAgentParams = appendParamToSlice(fenceAgentParams, paramName, paramVal)
} else if paramVal != parameterActionValue {
	logger.Info("FAR doesn't support any other action than reboot", "action", paramVal)
}
```